examples/react-alt/js/app.jsx

```diff
@@ -0,0 +1,176 @@
+/*jshint quotmark:false */
+/*jshint white:false */
+/*jshint trailing:false */
+/*jshint newcap:false */
+/*global React, Router*/
+var app = app || {};
+
+(function () {
+	'use strict';
+
+	app.ALL_TODOS = 'all';
+	app.ACTIVE_TODOS = 'active';
+	app.COMPLETED_TODOS = 'completed';
+	var TodoFooter = app.TodoFooter;
+	var TodoItem = app.TodoItem;
+	var TodoActions = app.todoActions;
+	var TodoStore = app.todoStore;
+
+	var ENTER_KEY = 13;
+
+	var TodoApp = React.createClass({
+		getInitialState: function () {
+			return TodoStore.getState();
```

Comment:
My thinking is that this perhaps obscures the example a bit by making it depart more from the current React example. One of my principles when writing this was that it should be as simple as possible to compare to the current React example.